### PR TITLE
PAYARA-2882 SSL session properties added (Payara 4)

### PR DESCRIPTION
**Pull request by Cousjava**
* PAYARA-2880 Made sslSessionTimeout and sslSessionCacheSize configurable

* PAYARA-2882 added configuration to admin console for SSL properties

Also renamed to tls instead of ssl

* PAYARA-2882 Use TLS not SSL

* Added copyright header

* Update string text

* PAYARA-2882 requested changes, some refactoring

* PAYARA-2882 Updated javadoc comment

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 4 please

File: nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/dom/Ssl.java [hunk 1]
```diff
@@ -312,5 +313,21 @@
     String getHandshakeTimeoutMillis();
 
     void setHandshakeTimeoutMillis(String timeoutMillis);
+    
+    /**
+     * How long before TLS sessions expire from the cache
```
Comment (MarkWareham) [inline nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/dom/Ssl.java]: Would be good to document which units are being worked with here

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 4 please

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 4 please

**Pull request merged**